stories/Containers/Navigation.stories.tsx

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
-import { Navigation, NavigationProps, NavigationFootnote } from '../../src';
-import { createStoryTitle } from '../Constants';
 import { Meta, Story } from '@storybook/react';
+import {
+    NavigationOld,
+    NavigationOldProps,
+    NavigationFootnote,
+} from '../../src';
+import { createStoryTitle } from '../Constants';
 
 export default {
-    title: createStoryTitle('Navigation'),
-    component: Navigation,
+    title: createStoryTitle('NavigationOld'),
+    component: NavigationOld,
     args: {
         footer: <NavigationFootnote url="" text="To Other Place" />,
     },
 } as Meta;
 
-export const Basic: Story<NavigationProps> = (args) => <Navigation {...args} />;
+export const Basic: Story<NavigationOldProps> = (args) => (
+    <NavigationOld {...args} />
+);
```

Comment:
We won't have duplicate navigations in the React-UI library. So just replace with your implementation